packages/firebase/index.d.ts

```diff
@@ -617,15 +617,14 @@ declare namespace firebase.firestore {
 
   export type LogLevel = 'debug' | 'error' | 'silent';
 
-  function setLogLevel(logLevel: LogLevel): void;
+  export function setLogLevel(logLevel: LogLevel): void;
```

Comment:
This is not needed as it is part of the `firebase` namespace which is exported at the end.

Reply:
Before I change this, do you know if this is causing a problem?

I would really like if these typings matched the ones in the firestore.d.ts file and having this extra 'export' seems like a step in that direction.

Reply:
This is fine. I looked and it seems to have no impact.